### PR TITLE
PPF-436 Create listeners

### PR DESCRIPTION
**Pull request by grubolsch**
### Added
This PR adds the listener when an integration is created, to automatically create the integration.

---
Ticket: https://jira.uitdatabank.be/browse/PPF-436

### CODE REVIEW

File: app/Keycloak/Client.php [hunk 1]
```diff
@@ -34,6 +34,6 @@ public static function createFromJson(Realm $realm, UuidInterface $integrationId
 
     public function getKeycloakUrl(string $baseUrl): string
     {
-        return sprintf('%s/admin/master/console/#/%s/clients/%s/settings', $baseUrl, $this->realm->internalName, $this->integrationId->toString());
+        return $baseUrl . sprintf('admin/master/console/#/%s/clients/%s/settings', $this->realm->internalName, $this->id->toString());
```
Comment (LucWollants) [inline app/Keycloak/Client.php:37]: Is it concatenation or interpolation? Mixing them feels a bit weird.

File: app/Keycloak/KeycloakServiceProvider.php [hunk 2]
```diff
@@ -48,5 +54,20 @@ public function register(): void
...
         });
+
+        $this->app->singleton(KeycloakClientRepository::class, function () {
+            return $this->app->get(EloquentKeycloakClientRepository::class);
+        });
+
+        $this->applyEvents();
+    }
+
+    private function applyEvents(): void
```
Comment (LucWollants) [inline app/Keycloak/KeycloakServiceProvider.php:65]: Maybe something like `bootstrapEventHandling` is better?

File: app/Keycloak/KeycloakServiceProvider.php [hunk 3]
```diff
@@ -48,5 +54,20 @@ public function register(): void
...
+    {
+        if (!$this->app->get(Config::class)->isEnabled) {
```
Comment (LucWollants) [inline app/Keycloak/KeycloakServiceProvider.php:67]: Nit: It is also possible to create the Config class on the stack on line 40 and then pass it in here. But both are fine.

File: app/Keycloak/Service/CreateClientHandler.php
Comment (LucWollants) [inline app/Keycloak/Service/CreateClientHandler.php:1]: This seems to be an empty file?

File: config/keycloak.php [hunk 4]
```diff
@@ -3,7 +3,7 @@
 declare(strict_types=1);
 
 return [
-    'enabled' => env('KEYCLOAK_ENABLED', false),
+    'enabled' => env('KEYCLOAK_ENABLE', false),
```
Comment (LucWollants) [inline config/keycloak.php:6]: I guess the old name was better?

**Review by LucWollants — APPROVED** [review]
I already approved, just some minor remarks

File: config/keycloak.php (hunk 4 above)
Comment (grubolsch) [reply config/keycloak.php]: Fine, but that does request an extra pr: https://github.com/cultuurnet/appconfig/pull/632

**Pull request merged**